### ISSUE TITLE
Restrict the access given to the GitHub App

### ISSUE THREAD

**Issue by ironnysh** [adamziel/playground-docs-workflow#38]
This project is mindblowing. I love it! 🥇

However, I feel like the GitHub authorization step is going to be a major barrier:
<img width="631" alt="image" src="https://github.com/adamziel/playground-docs-workflow/assets/63248335/6c79e14f-33d7-4559-a85d-1b7e18032830">

1. It appears to be granting access to you, @adamziel :-)
2. The scope (**read and write all public and private repository data**, and **access to manage organization attributes and organization-owned resources including projects, invitations, team memberships and webhooks. This scope also grants the ability to manage projects owned by users.**) is alarming.

Is it possible to use [GitHub's fine-grained personal access tokens](https://github.blog/2022-10-18-introducing-fine-grained-personal-access-tokens-for-github/)?

**Comment by adamziel** [comment]
Good point! @ironnysh :-) That was my temporary solution. We should have a fine-grained WP.org app. :wave: @dd32, who should I ask to get it set up?
